Dockerfile

```diff
@@ -1,24 +1,7 @@
-FROM ubuntu:16.04 as builder
+ARG BUILDER_IMAGE
+FROM ${BUILDER_IMAGE} as builder
 
-RUN apt-get update \
- && apt-get install -y \
-    git build-essential software-properties-common sudo \
- && add-apt-repository -y ppa:ubuntu-toolchain-r/test \
- && apt-get update \
- && apt-get install -y gcc-7 \
- && cd /usr/bin/ \
- && rm gcc \
- && ln -s gcc-7 gcc \
- && git clone https://gerrit.fd.io/r/vpp /vpp
-
-ADD build.env /
-
-RUN . /build.env \
- && cd /vpp \
- && git pull \
- && git checkout ${VPP_COMMIT} \
- && UNATTENDED=y make install-dep bootstrap pkg-deb \
- && cd build-root \
+RUN cd /vpp/build-root \
  && bash -c "tar -czvf vpp-debs.tar.gz {vpp,vpp-plugins,vpp-lib}_$(git describe --tags | sed -e s/-/~/2 -e s/^v//)_amd64.deb"
```

Comment:
do you need anything from build.env?

Reply:
not anymore from this file, that's why not added